scalafix-tests/output/src/main/scala/test/escapeHatch/EscapeHatchPatches.scala

```diff
@@ -0,0 +1,8 @@
+package test.escapeHatch
+
+import scala.collection.immutable // scalafix:ok RemoveUnusedImports
+
+import scala.collection.mutable.{ // scalafix:ok RemoveUnusedImports
```

Comment:
That's likely because removing `Map` or `Set` independently removes the `{` which is escaped. If the comment only applies to `import scala` I suspect it won't work

Reply:
This is good then, it makes it really difficult to fall on the non-supported case